mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -326,7 +326,8 @@ JUGG:
 	Mobile:
 		Speed: 71
 		TurnSpeed: 5
-		RequiresCondition: !empdisable && undeployed && !being-captured
+		RequiresCondition: undeployed
+        PauseOnCondition: empdisable || being-captured
```

Comment:
Tabs instead of spaces, please (in the other places, too).

Reply:
Fixed.